File: seed/php-sdk/examples/src/File/FileClient.php
<?php

namespace Seed\File;

use Seed\File\Notification\NotificationClient;
use Seed\File\Service\ServiceClient;
use Seed\Core\Client\RawClient;

class FileClient
{
    /**
     * @var NotificationClient $notification
     */
    public NotificationClient $notification;

    /**
     * @var ServiceClient $service
     */
    public ServiceClient $service;

    /**
     * @var RawClient $client
     */
    private RawClient $client;

    /**
     * @param RawClient $client
     */
    public function __construct(
        RawClient $client,
    ) {
        $this->client = $client;
        $this->notification = new NotificationClient($this->client);
        $this->service = new ServiceClient($this->client);
    }
}

File: seed/csharp-sdk/examples/no-custom-config/src/SeedExamples/File/FileClient.cs
using SeedExamples.Core;

namespace SeedExamples.File;

public partial class FileClient
{
    private RawClient _client;

    internal FileClient(RawClient client)
    {
        _client = client;
        Notification = new SeedExamples.File.Notification.NotificationClient(_client);
        Service = new SeedExamples.File.ServiceClient(_client);
    }

    public SeedExamples.File.Notification.NotificationClient Notification { get; }

    public SeedExamples.File.ServiceClient Service { get; }
}
